### ISSUE TITLE
Pickaxes seem extremely ineffective against any block

### ISSUE THREAD

**Issue by Ephememory** [Facepunch/sbox-corewars#24]

**Comment by Ephememory** [comment]
> There's 3 levels of pickaxe, each one getting stronger and stronger. However, it could be the case that the default damage multiplier is too low.

Yeah, I tested with tier 2, not 3 yet. Dunno the item name to spawn in iron or pickaxes, haven't looked yet. I tried on fungus, wood, and glass and it seemed too slow even for tiers 1 and 2.

**Comment by kurozael** [comment]
I've adjusted them a bit but I think they still need some work. For now I'll close it and we'll re-evaluate on the next playtest.
